### DISCUSSION TITLE
Shinewifi-X no "set ip" option to edit

### DISCUSSION THREAD

**Discussion by rootje** [johanmeijer/grott#240] [Q&A] [answered]
Hi, I can't figure out what I am doing wrong, but I don't have an option to change the "set ip" to another value. It's not showing up in my settings page. What am I doing wrong, or do have to look somewhere else? I have a ShineWifi-X datalogger, so it should be compatible.

Please help!

![image](https://user-images.githubusercontent.com/4248442/203162174-6dd06744-3378-43ea-86cf-2b5471858740.png)

**Comment by robupham** [comment, marked as answer]
Over on the right you see  “All device” - click that and you can select the Data Logger.
Then change the settings for that, not the inverter.

**Reply by rootje** [reply]
Think I should go for some new glasses. Sometimes things can be so easy. Thank you so much for answering!
